contracts/src/v0.8/ccip/test/rmn/RMNRemote/RMNRemote.globalAndLegacyCurses.t.sol

```diff
@@ -1,11 +1,11 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.24;
 
-import {GLOBAL_CURSE_SUBJECT, LEGACY_CURSE_SUBJECT} from "../../../rmn/RMNRemote.sol";
+import {GLOBAL_CURSE_SUBJECT} from "../../../rmn/RMNRemote.sol";
 import {RMNRemoteSetup} from "./RMNRemoteSetup.t.sol";
 
-contract RMNRemote_global_and_legacy_curses is RMNRemoteSetup {
-  function test_global_and_legacy_curses_success() public {
+contract RMNRemote_global_curses is RMNRemoteSetup {
+  function test_global_curses_success() public {
```

Comment:
Please use the new test naming convention for anything you touch. It's best to update the entire file once you touch it to maintain consistency.